ml-agents/mlagents/trainers/tf/model_serialization.py

```diff
@@ -120,20 +114,18 @@ def export_policy_model(
 
 
 def _make_frozen_graph(
-    settings: SerializationSettings, graph: tf.Graph, sess: tf.Session
+    brain_name: str, graph: tf.Graph, sess: tf.Session
```

Comment:
brain is deprecated terminology. Can we replace with behavior_name here ?

Reply:
Are we not using brain_name at all now? This argument name comes all the way from trainer -> saver -> model_serialization and I still see brain_name being used in trainer/policy/many elsewhere. 
Where should I start changing it? Just this function or should I change them all?

Reply:
brain does not exist anymore, we removed it from all public APIs. We do not need to change it everywhere, but we should modify them if the we change the code. (so just on the methods that you modify)

Reply:
Okay I changed it for all the saver and serialization files. Still think we should change them all since it's kind of confusing to refer to the same thing with different names in different places.